allennlp_models/common/model_card.py

```diff
@@ -0,0 +1,267 @@
+import os
+import logging
+from dataclasses import dataclass
+from typing import Optional, Union, Mapping, Dict
+
+from allennlp.models import Model
+from allennlp.common.checks import ConfigurationError
+
+PRETRAINED_MODELS: Dict = {}
+STORAGE_LOCATION = "https://storage.googleapis.com/allennlp-public-models/"
+
+logger = logging.getLogger(__name__)
+
+
+def get_pretrained_models():
+    pretrained_models = {}
+    for key, val in PRETRAINED_MODELS.items():
+        pretrained_models[key] = val.to_dict()
+    return pretrained_models
+
+
+def get_description(model_class):
+    """
+    Returns the model's description from the docstring.
+    """
+    return model_class.__doc__.split("# Parameters")[0].strip()
+
+
+@dataclass
+class ModelCardInfo(object):
```

Comment:
Python 3 nit: inheriting from object is unnecessary in python 3.
```suggestion
class ModelCardInfo:
```